elastic/kibana
# [Fleet] Fix package icon size when browsing from list to details page

## Summary

Fix a bug introduced in #165030 

The package icon size was not correct when browsing from the integration list page to the details one.

This was due to a bad use of emotion `css`. That PR fix that


## How to test

Visit the integration list => search for ec2 => go to ec2 ensure the icon size is correct

## Before the fix 

![Zrzut ekranu 2023-08-31 o 21 18 37](https://github.com/elastic/kibana/assets/1336873/b4aceebb-1cd8-4492-8c45-a4bd43a2441f)
